connectors/sources/mysql.py

```diff
@@ -18,6 +19,8 @@
 from connectors.sources.generic_database import WILDCARD, configured_tables, is_wildcard
 from connectors.utils import CancellableSleeps, RetryStrategy, retryable
 
+SPLIT_BY_COMMA_OUTSIDE_BACKTICKS_PATTERN = r"`(?:[^`]|``)+`|\w+"
```

Comment:
not super important, but it's good practive to compile it:

```
SPLIT_TABLES = re.compile(r"`(?:[^`]|``)+`|\w+")
```

and then:

```
SPLIT_TABLES.findall(tables_string)
```

Reply:
Right, thank you for the hint! Adjusted it.